dialogue/TownNPCs/Mocha/Any_Mocha_StandingOnCounter.json

```diff
@@ -16,12 +16,18 @@
       "¿Podrías no hacer eso?",
       "De verdad.\nNo me apetece que los clientes entren viendo este panorama."
     ],
+    "pl": [
+      "...\n...\nPrzepraszam?",
+      "Czy mógłbyś proszę, tak, nie robić?",
```

Comment:
The commas here are a little bit unnecessary

Reply:
I put in the commas since the other translations do as well, since they make the dialogue feel awkward.

Reply:
Oh, right. Didn't notice that 😅